mxbluesky/devices/generic.py

```diff
@@ -1,5 +1,7 @@
 from ophyd import Component as Cpt
-from ophyd import Device, EpicsMotor, EpicsSignal, EpicsSignalRO, PVPositionerIsClose
+from ophyd import Device, EpicsMotor, EpicsSignal, EpicsSignalRO
+from ophyd.status import SubscriptionStatus
+from ophyd import DynamicDeviceComponent as DDC
```

Comment:
FYI you can do this on the same line as the other ophyd imports, not sure which black prefers though.

Reply:
Makes sense to import together, fixed in https://github.com/NSLS2/lsdc/pull/399/commits/016572acc673dfedca454f166725235945b69d11